sqlglot/parser.py

```diff
@@ -2567,7 +2567,10 @@ def _parse_returns(self) -> exp.ReturnsProperty:
 
     def _parse_describe(self) -> exp.Describe:
         kind = self._match_set(self.CREATABLES) and self._prev.text
-        style = self._match_texts(("EXTENDED", "FORMATTED", "HISTORY")) and self._prev.text.upper()
+        style = (
+            self._match_texts(("ANALYZE", "EXTENDED", "FORMATTED", "HISTORY"))
```

Comment:
Let's make this a class-level set constant and reference that instead.

Reply:
Makes sense. Thanks for the quick review!